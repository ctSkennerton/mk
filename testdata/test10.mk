# Run an external command.
# Assume that go is installed.
<|go env

test10.mk.o:
	$GCCGO test10.mk.c
