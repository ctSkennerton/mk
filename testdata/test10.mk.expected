test10.mk.o: gccgo test10.mk.c
